Dockerfile

```
@@ -5,8 +5,10 @@ FROM python:${PYTHON_VERSION}-${VARIANT} as base
 
 ENV PYTHONUNBUFFERED=1
 
-RUN python -m ensurepip --upgrade
-RUN pip install --upgrade setuptools
+COPY ./requirements.txt ./
+
+RUN python -m ensurepip
+RUN pip install -r requirements.txt
```

```suggestion
RUN python -m ensurepip
RUN pip install -r requirements.txt && rm requirements.txt
```

1. Let's not leave not needed files behind
2. Do we still need `python -m ensurepip --upgrade`?

---

We're just ensuring that pip exists. Should we remove that?